salt/cloud/clouds/vultrpy.py

```diff
@@ -282,6 +356,21 @@ def create(vm_):
         )
         return False
 
+    firewall_group_id = config.get_cloud_config_value(
+        'firewall_group_id', vm_, __opts__, search_global=False, default=None,
+    )
+
+    if firewall_group_id and str(firewall_group_id) not in avail_firewall_groups():
+        log.error('Your Vultr account does not have a firewall group with ID %s', str(firewall_group_id))
+        return False
+
+    key_list = ssh_key_ids.split(',')
```

Comment:
if `ssh_key_ids` is None this will fail with an AttributeError.

Reply:
Good point! I will fix this!